pkg/api/v20160330/validate.go

```diff
@@ -71,6 +71,15 @@ func (l *LinuxProfile) Validate() error {
 
 // Validate implements APIObject
 func (a *Properties) Validate() error {
+	if a.OrchestratorProfile == nil {
+		return fmt.Errorf("missing OrchestratorProfile")
+	}
```

Comment:
So you're suggesting that when `a.OrchestratorProfile == nil` that we do something like this?:

```
a.OrchestratorProfile = &OrchestratorProfile{OrchestratorType: "DCOS"}
```